staff/carlos-matilla/precourse/bingo/index.js

```diff
@@ -0,0 +1,148 @@
+// Variables
+var carton = [];
+var bombo = [];
+var nombre = prompt('Cual es tu nombre?');
+
+//Acumuladores
+var lin1 = 0;
+var lin2 = 0;
+var lin3 = 0;
+var turnos = 0;
+
+//Funcion que crea un array de numeros aleatorios del 1 al 99
+function getBombo() {
+    for(var i = 1; i < 100; i++) {
+        function genBom() {
+            var numAle =  Math.floor(Math.random() * 99) + 1;
+            if(bombo.indexOf(numAle) == -1){ 
+                return bombo.push(numAle);
+            } else {
+                return genBom();
+            }
+        }
+        genBom(); 
+    }
+}
+
+//Funcion principal
+function bingo(){
+    //Funcion que crea un carton de 15 numeros aleatorios
+    function getCarton() {
+        for(var i = 1; i < 16; i++) {
+            function genCarton() {
+                var numAle =  Math.floor(Math.random() * 99) + 1;
+                if(carton.indexOf(numAle) == -1){ 
+                      return carton.push(numAle);
+                } else {
+                        return genCarton();
+                }
+            }   
+            genCarton(); 
+        }
+    }
+    getCarton();
+
+    console.log('')
+    console.log('Hola ' + nombre)
+    console.log('')
+    console.log('Este es el carton que se te ha asignado:')
+    console.log('')
+    console.log(carton.slice(0,5))
+    console.log(carton.slice(5,10))
+    console.log(carton.slice(10))
+    console.log('')
```

Comment:
Transform this in one console.log() with \n. For example put this in the chrome console: console.log("\nVictori \nMurillo") and see the result.

Reply:
Thank you Mate, I'll solve the problem now!!